com.unity.render-pipelines.universal/Runtime/RendererFeatures/ScreenSpaceAmbientOcclusion.cs

```diff
@@ -376,42 +372,25 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
                     // If true, SSAO pass is inserted after opaque pass and is expected to modulate lighting result now.
                     if (m_CurrentSettings.AfterOpaque)
                     {
-                        CoreUtils.SetRenderTarget(
-                            cmd,
-                            m_Renderer.cameraColorTargetHandle,
-                            RenderBufferLoadAction.Load,
-                            RenderBufferStoreAction.Store,
-                            m_Renderer.cameraDepthTargetHandle,
-                            RenderBufferLoadAction.Load,
-                            RenderBufferStoreAction.Store,
-                            ClearFlag.None,
-                            Color.clear
-                        );
-                        cmd.DrawMesh(RenderingUtils.fullscreenMesh, Matrix4x4.identity, m_Material, 0, (int)ShaderPasses.AfterOpaque);
+                        bool isRenderToBackBufferTarget = !cameraData.isSceneViewCamera; //&& !cameraData.isDefaultViewport;
```

Comment:
Need to figure out this conditional

Reply:
Should we do check similar to postFX finalblit? I don't think SSAO after opaque always render to backbuffer. It could still render to intermediate render target right?